StrictEmit/ILGeneratorExtensions.Objects.cs

```diff
@@ -96,16 +97,17 @@ public static void EmitSetObject([NotNull] this ILGenerator il, [NotNull] Type o
         /// <summary>
         /// Tests whether an object reference (type <strong>O</strong>) is an instance of a particular class.
         /// </summary>
-        /// <typeparam name="T">The type.</typeparam>
+        /// <typeparam name="T">The type to emit the instance of instruction.</typeparam>
         /// <param name="il">The generator where the IL is to be emitted.</param>
         [PublicAPI]
-        public static void EmitIsInstance<T>([NotNull] this ILGenerator il) => il.EmitIsInstance(typeof(T));
+        public static void EmitIsInstance<T>([NotNull] this ILGenerator il)
+            => il.EmitIsInstance(typeof(T));
 
         /// <summary>
         /// Tests whether an object reference (type <strong>O</strong>) is an instance of a particular class.
         /// </summary>
         /// <param name="il">The generator where the IL is to be emitted.</param>
-        /// <param name="typeToCheck">The type.</param>
+        /// <param name="typeToCheck">The type of object to emit if it is an instance of.</param>
```

Comment:
Rather clunky. Perhaps "The object type to test against"?

Reply:
i just noticed that right above you have "The type to emit the instance of instruction", should i use that instead?